images/plbase/plbase-install.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -ex
 
+# If you need to rebuild this image without actually changing anything,
+# add a dot to the following line:
+# .
```

Comment:
This is very stupid, but it works 🤷 is there any better way to do this?

Reply:
Not that I know of 😅